Grasshopper_UI/Render/CreatePreviewMesh.cs

```diff
@@ -0,0 +1,192 @@
+using BH.oM.Geometry;
+using System.Drawing;
+using BH.Engine.Rhinoceros;
+
+namespace BH.UI.Grasshopper
+{
+    public static partial class Render
+    {
+        /***************************************************/
+        /**** Public Methods  - Interfaces              ****/
+        /***************************************************/
+
+        public static RHG.Mesh ICreatePreviewMesh(this object obj, RHG.MeshingParameters parameters)
+        {
+            if (obj == null)
+            {
+                return null;
+            }
+            if (parameters == null)
+                parameters = RHG.MeshingParameters.Default;
+
+            try
+            {
+                return CreatePreviewMesh(obj as dynamic, parameters);
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+        }
+
+        /***************************************************/
+
+        public static RHG.Mesh CreatePreviewMesh(this List<object> geometry, RHG.MeshingParameters parameters)
+        {
+            RHG.Mesh mesh = new RHG.Mesh();
+            foreach (object geo in geometry)
+            {
+                RHG.Mesh itemMesh = CreatePreviewMesh(geo as dynamic, parameters);
+                if (itemMesh != null)
+                    mesh.Append(itemMesh);
+            }
+            return mesh;
+        }
+
+
+        /***************************************************/
+        /**** Private Methods  - Fallback               ****/
+        /***************************************************/
+
+        private static RHG.Mesh CreatePreviewMesh(this object fallback, RHG.MeshingParameters parameters)
+        {
+            // fallback in case no method is found for the provided runtime type
+            return null;
+        }
+
+
+        /***************************************************/
+        /**** Public Methods  - Surfaces                ****/
+        /***************************************************/
+
+        public static RHG.Mesh CreatePreviewMesh(RHG.Extrusion surface, RHG.MeshingParameters parameters)
+        {
+            return CreatePreviewMesh(surface.ToBrep(), parameters);
+        }
+
+        /***************************************************/
+
+        public static RHG.Mesh CreatePreviewMesh(RHG.NurbsSurface surface, RHG.MeshingParameters parameters)
+        {
+            return CreatePreviewMesh(surface.ToBrep(), parameters);
+        }
+
+        /***************************************************/
+
+        public static RHG.Mesh CreatePreviewMesh(RHG.Brep brep, RHG.MeshingParameters parameters)
+        {
+            RHG.Mesh[] array = RHG.Mesh.CreateFromBrep(brep, parameters);
+            if (array == null)
+            {
+                return new RHG.Mesh();
+            }
+            if (array.Length == 1)
+            {
+                return array[0];
+            }
+            RHG.Mesh mesh = new RHG.Mesh();
```

Comment:
```suggestion
            if (array == null)
                return new RHG.Mesh();

            if (array.Length == 1)
                return array[0];

            RHG.Mesh mesh = new RHG.Mesh();
```

Again just giving some spacing around the different checks/blocks. Happy to keep the brackets but if so can we then put a blank line between each different `if` block instead.